external/fv3fit/fv3fit/emulation/thermobasis/emulator.py

```diff
@@ -343,17 +352,22 @@ def load(cls, path: str):
         return model
 
 
+def get_regularizer(config: Config) -> Optional[tf.keras.regularizers.Regularizer]:
```

Comment:
nit: Elsewhere you use a pattern of having properties on Config, e.g. `config.target.loss`. Does it make sense to have `config.regularizer` for consistency? This can be always defined if you use a regularizer with default norm of zero.

Reply:
It's true this is inconsistent, but there is an existing pattern of `get_model(config)` below. I'm not sure which is better in general. In this case `get_regularizer` is mostly a helper for `get_model`, so I'm not sure I want to make it more visible by putting it on the Config directly.

Reply:
Let's keep it like this for this PR. For your future consideration, a benefit of using `config.regularizer` instead of `get_regularizer(config)` is that the former allows injection of new configs into functions of this package, whereas the latter requires you to update `get_regularizer` whenever you update `config`.

Honestly from what you're saying it sounds like there should be a `config.get_model` method. In the top-level infrastructure there is a `get_training_function` method, but that's because it takes a configuration string as an input, which isn't an object we can control and add methods to.